shell/platform/glfw/glfw_event_loop.cc

```diff
@@ -73,8 +73,10 @@ void GLFWEventLoop::WaitForEvents(std::chrono::nanoseconds max_wait) {
     const auto duration_to_wait = std::chrono::duration_cast<Seconds>(
         std::min(next_wake - now, max_wait));
 
-    if (duration_to_wait.count() > 0.0) {
-      ::glfwWaitEventsTimeout(duration_to_wait.count());
+    double wait_count = std::min(duration_to_wait.count(), (double)LONG_MAX);
```

Comment:
This would need to be a `static_cast`, not a C-style cast, per Google style guide.